src/interface/rest/server/resource/k8scluster.go

```diff
@@ -625,23 +625,21 @@ func RestPostK8sClusterDynamic(c echo.Context) error {
 // @Description **Example request body:**
 // @Description ```json
 // @Description {
-// @Description   "namePrefix": "across",
+// @Description   "namePrefix": "eu-cluster",
```

Comment:
```suggestion
// @Description   "namePrefix": "across",
```

The term is based on very intentional selection. :)

Reply:
@seokho-son 

I figured out that Tencent TKE requires the TKE_QCSRole service role.
I just created this role, and since we share the same Tencent account, you should be able to use it without any additional setup.
As far as I know, there is no separate guide for this yet.

<img width="1211" height="909" alt="image" src="https://github.com/user-attachments/assets/beebbfff-b2c2-4cab-bbb7-8e1922a7648b" />